homekit/StateManager.js

```diff
@@ -250,105 +325,111 @@ module.exports = (device, platform) => {
 
 				if (!active || mode !== 'DRY') {
 					log.easyDebug(device.name, '(GET) - Dry Current Dehumidifier State: INACTIVE')
+
 					callback(null, Characteristic.CurrentHumidifierDehumidifierState.INACTIVE)
 				} else {
 					log.easyDebug(device.name, '(GET) - Dry Current Dehumidifier State: DEHUMIDIFYING')
+
 					callback(null, Characteristic.CurrentHumidifierDehumidifierState.DEHUMIDIFYING)
 				}
 			},
 
 			TargetHumidifierDehumidifierState: (callback) => {
-				log.easyDebug(device.name, '(GET) - Target Dehumidifier State: DEHUMIDIFIER')
+				log.easyDebug(device.name, '(GET) - HeaterCooler State: ?')
+
 				callback(null, Characteristic.TargetHumidifierDehumidifierState.DEHUMIDIFIER)
 			},
 
 			DryRotationSpeed: (callback) => {
 				const fanSpeed = device.state.fanSpeed
 
-				log.easyDebug(device.name, '(GET) - Dry Rotation Speed is:', fanSpeed + '%')
+				log.easyDebug(device.name, '(GET) - Dry Rotation Speed:', fanSpeed + '%')
+
 				callback(null, fanSpeed)
 			},
 
 			DrySwing: (callback) => {
 				const swing = device.state.verticalSwing
 
-				log.easyDebug(device.name, '(GET) - Dry Swing is:', swing)
+				log.easyDebug(device.name, '(GET) - Dry Swing:', swing)
+
 				callback(null, Characteristic.SwingMode[swing])
 			},
 
 			// ROOM SENSOR
 			MotionDetected: (callback) => {
 				const motionDetected = device.state.motionDetected
+				log.easyDebug(device.name, '(GET) - Motion Detected:', motionDetected)
 
-				log.easyDebug(device.name, '(GET) - Motion Detected is:', motionDetected)
 				callback(null, motionDetected)
 			},
 
 			StatusLowBattery: (callback) => {
 				const lowBattery = device.state.lowBattery
+				log.easyDebug(device.name, '(GET) - Status Low Battery:', lowBattery)
 
-				log.easyDebug(device.name, '(GET) - Status Low Battery is:', lowBattery)
 				callback(null, Characteristic.StatusLowBattery[lowBattery])
 			},
 
 			// HORIZONTAL SWING
 			HorizontalSwing: (callback) => {
 				const horizontalSwing = device.state.horizontalSwing
+				log.easyDebug(device.name, '(GET) - Horizontal Swing:', horizontalSwing)
 
-				log.easyDebug(device.name, '(GET) - Horizontal Swing is:', horizontalSwing)
 				callback(null, horizontalSwing === 'SWING_ENABLED')
 			},
 
 			// AC LIGHT
 			LightSwitch: (callback) => {
 				const light = device.state.light
+				log.easyDebug(device.name, '(GET) - AC Light:', light ? 'ON' : 'OFF')
```

Comment:
LightSwitch is common...
```suggestion
				log.easyDebug(device.name, '(GET) - Light:', light ? 'ON' : 'OFF')
```